goc/share/oasis-batch-worker

```diff
@@ -39,8 +39,10 @@ if [ $vo ]; then
        echo "Update initiated: $now">/tmp/update.details
        now=`date +%s`
        echo "Update unix time: $now">>/tmp/update.details
-       if [ ! -f $cert ] ; then
-	   echo "No proxy found for user $user" >>/tmp/update.details
+       if [ -n "$CILOGONID ]; then
+           echo "via ssh from $CILOGONID" >>/tmp/update.details
+       elif [ ! -f $cert ] ; then
```

Comment:
I know this wasn't your change from this PR, but i'll note it here that all the `[ ... ]` tests with unquoted variables would be safer written with `[[ ... ]]`, which does not do word splitting and glob pathname expansion.

```suggestion
       elif [[ ! -f $cert ]] ; then
```

Reply:
I'll fix those unquoted variables.